caps/types.go

```diff
@@ -60,6 +63,24 @@ func (t *BoolFileType) Sanitize(c *Capability) error {
 	return nil
 }
 
+// SecuritySnippet for bool-file capability type.
+// Consumers gain permission to read, write and lock the designated file.
+func (t *BoolFileType) SecuritySnippet(c *Capability, securitySystem string) (string, error) {
+	switch securitySystem {
+	case SecurityApparmor:
+		// TODO: switch to the absolute path later
+		path := c.Attrs["path"]
+		// Allow read,write and lock on the file designated by the path.
+		return fmt.Sprintf("%s rwl,\n", path), nil
+	case SecuritySeccomp:
+		return "", nil
```

Comment:
From what I read so far seccomp defaults are sufficient for bool-file (open, read, write are all there). @jdstrand can you confirm this please?